master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md

```diff
@@ -77,7 +75,7 @@ export KUBE_MASTER_OS_DISTRIBUTION=debian
 curl -sS https://get.k8s.io | bash
 ```
 
-##### 2) Install Calico using the etcdless manifest
+##### 2) Install Calico using the Kubenetes datastore manifest
```

Comment:
I'm not sure we want to be calling it the Kubernetes datastore driver.  I think "Kubernetes datastore manifest" is sufficient.

The "Kubernetes datastore driver" is a specific piece of code.